content/en/docs/rate-limits.md

```diff
@@ -3,131 +3,148 @@ title: Rate Limits
-faster than it resets on its own.
-
-Note that most hosting providers don't need rate limit increases, because
-there's no limit on the number of distinct registered domains for which you can issue.
-So long as most of your customers don't have more than 2,000 subdomains on a
-registered domain, you most likely do not need an increase. See our [Integration Guide](/docs/integration-guide) for more advice.
-
-# <a id="clearing-pending"></a>Clearing Pending Authorizations
-
-If you have a large number of pending authorization objects and are getting a
-Pending Authorizations rate limiting error, you can trigger a validation attempt for those
-authorization objects by submitting a JWS-signed POST to one of its challenges, as
-described in the
-[ACME spec](https://tools.ietf.org/html/rfc8555#section-7.5.1).
-The pending authorization objects are represented by URLs of the form
-`https://acme-v02.api.letsencrypt.org/acme/authz/XYZ`, and should show up in your
-client logs. Note that it doesn't matter whether validation succeeds or fails.
-Either will take the authorization out of 'pending' state. If you do not
-have logs containing the relevant authorization URLs, you need to wait for the
-rate limit to expire. As described above, there is a sliding window, so this may
-take less than a week depending on your pattern of issuance.
-
-Note that having a large number of pending authorizations is generally the
-result of a buggy client. If you're hitting this rate limit frequently you
-should double-check your client code.
+# Foreword
+
+To ensure fair usage, Let's Encrypt enforces rate limits that we believe to be
+reasonable for most scenarios. Our rate limits were designed to facilitate unhampered
+certificate renewals and progressive certificate usage by large organizations. If you
+are integrating Let's Encrypt into your operations, please review our [Integration Guide](/docs/integration-guide)
+to ensure smooth operation. In particular, we recommend using one ACME account for many
+customers. If you are developing an ACME client, debugging the usage of one, or configuring
+an automated system, please take advantage of our [staging environment](/docs/staging-environment),
+which is governed by higher rate limits and issues false certificates.
+
+We have no way of temporarily overriding the rate limits. For very large providers who are
+encountering rate limits due to sheer volume, we provide a [rate limiting form](https://goo.gl/forms/plqRgFVnZbdGhE9n1)
+that can be used to request higher rate limits. Since these requests can take several weeks
+to process, the form is not suitable for addressing temporary situations that will
+automatically be resolved when any encountered rate limits no longer apply.
+
+Revoked certificates still count against our rate limits and remain in the public record.
+All issued certificates can be found using [crt.sh](https://crt.sh/), which uses public
```

Comment:
I had this discussion with @osirisinferi a while back. 😄 Considering who is the sponsor of [Let's Encrypt's transparency logs ](https://letsencrypt.org/docs/ct-logs/), the fact that the all of the [root and intermediate certificate links](https://letsencrypt.org/certificates/) are to crt.sh (per my own effort), and that crt.sh was already mentioned in the previous version of the page, I didn't feel like offering other options was really necessary. I'm open to the possibility though if it seems prudent.

Reply:
Note that crt.sh does actually sometimes suffer from a backlog due to performance reasons. That said, so did Google one day when crt.sh wasn't uptodate.. So I thought it wasn't a backlog issue, as crt.sh and Google didn't have a recent certificate.. But then someone came with a *third* CT log aggregator which *did* have a recent cert on file..
So, crt.sh isn't ideal due to sometimes performance reasons, nor is Google apparently. I'll try to see if I can find the name of that third option.

Reply:
If the point here is just that "it's all still in a the public record", it might be best to just reference the Certificate Transparency concept as a whole (that there are many distributed logs used to track that Certificate Authorities are following the rules) rather than making it seem like some particular log aggregator is the be-all reference point. The only real "official site" I know for it is https://www.certificate-transparency.org which may not really handle well explaining to the "general public" (and parts of it haven't really been updated to explain that it's now standard). Maybe it should just link to https://letsencrypt.org/docs/ct-logs/ (and maybe that page itself could use some updates at some point too)?

Reply:
Thanks for the feedback guys. Good to see you. 🙂

I know that crt.sh can be flaky sometimes, but it is far and away the most commonly-referenced aggregator mentioned on both Let's Encrypt's website and community. It's simply being mentioned here as **a** tool in the same way it was in the existing page, not as an advertisement for **the** tool. The utility provided by crt.sh here is more than just to show the theoretical concept of certificate transparency though. It is also meant to lower the defenses of certificate-seekers in terms of information sharing by letting them see with their own eyes that their certificate activities are not private affairs.

I fully concur about the link switch,  @petercooperjr. There's no loss since the current link is already on /docs/ct-logs anyhow.